inc/namespace.php

```diff
@@ -71,3 +74,44 @@ function set_gaussholder_image_sizes( array $sizes ) : array {
 function override_aws_rekognition_aws_client( $client, array $params ) : RekognitionClient {
 	return Platform\get_aws_sdk()->createRekognition( $params );
 }
+
+/**
+ * Add rekognition keywords to the search index
+ *
+ * @param array $post_data
+ * @param integer $post_id
+ * @return array
+ */
+function add_rekognition_keywords_to_search_index( array $post_data, int $post_id ) : array {
+
+	// This filter is called for all post types, so only add data to our "attachemnts" post type.
+	if ( $post_data['post_type'] !== 'attachment' ) {
+		return $post_data;
+	}
+
+	$post_data['alt'] = get_post_meta( $post_id, '_wp_attachment_image_alt', true );
+
+	$labels = get_attachment_labels( $post_id );
+	$labels = array_filter( $labels, function ( array $label ) : bool {
+		return $label['Confidence'] > 50;
```

Comment:
We could make this confidence value configurable at some point. Seems like a sensible default right now though.